### ISSUE TITLE
Pushing new transifex resource on config update for existing file

### ISSUE THREAD

**Issue by joekur** [lumoslabs/txgh#69]
If you have an already existing translation file in your repo, and then you push a commit to add that to transifex by adding it in `.tx/config`, txgh will not push that resource to transifex because the github webhook does not have the translation file as part of the added/modified files.

Is this something that sounds worth supporting? The easiest way to do this would be to re-sync every resource. Otherwise you would have to diff the `.tx/config` against the last version of it to see which resources were actually added.

**Comment by camertron** [comment]
Yeah, unfortunately that's a use case txgh doesn't support super well. All we'd really have to do is listen for changes to .tx/config in addition to files that match the `file_filter`s. I think I filed an issue for that internally, but never got around to implementing it.

**Comment by camertron** [comment]
I did however write a couple of scripts for us internally to do this. I had originally wanted to create a txgh CLI client, but it was pretty low priority. [This script](https://gist.github.com/camertron/22998d689b30e0566f4f48b454c5dbf7) uploads all resources for the master branch and [this script](https://gist.github.com/camertron/6f2de68b97dd1635fba6980746d5d6ee) validates your setup is correct. Obviously these are tailored for our needs and some work would need to be done to make them applicable to a wider user base, but maybe they'll work for you :)
